packages/round-manager/src/features/round/ViewRoundPage.tsx

```diff
@@ -135,6 +136,17 @@ export default function ViewRoundPage() {
                     {formatDate(round?.roundEndTime) || "..."}
                   </p>
                 </div>
+
+                <div className="flex justify-end grow relative">
+                  <div className="text-right absolute bottom-0">
+                    <p className="text-xs mb-1">Copy link to round application</p>
+                    <CopyToClipboardButton 
+                      textToCopy={window.location.href}
```

Comment:
should the text to copy be `https://granthub.gitcoin.co/#/chains/${chain_id}/rounds/ ${round_contract_address}`?